autogpt/commands/command.py

```diff
@@ -66,30 +73,39 @@ def register(self, cmd: Command) -> None:
                 f"Command '{cmd.name}' already registered and will be overwritten!"
             )
         self.commands[cmd.name] = cmd
+        for command_name_alias in cmd.name_alias:
+            if command_name_alias == cmd.name:
+                continue
+            self.commands[command_name_alias] = cmd
```

Comment:
- Each alias will be sent as a separate command to the LLM, duplicating descriptions and signatures. This results in increased use of the context window and could confuse the LLM. Can we include aliases in the same command output sent to the LLM?
- I'm curious how this works with the new Open AI function calling feature.

Reply:
Reverted this; I think duplicating the command definition in the prompt won't add much value. It's more about catching hallucinations.